server/src/main/java/org/elasticsearch/action/admin/cluster/stats/RemoteClusterStatsResponse.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.cluster.stats;
+
+import org.elasticsearch.action.support.nodes.BaseNodesResponse;
+import org.elasticsearch.cluster.ClusterName;
+import org.elasticsearch.cluster.health.ClusterHealthStatus;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Trimmed down cluster stats response for reporting to a remote cluster.
+ */
+public class RemoteClusterStatsResponse extends BaseNodesResponse<ClusterStatsNodeResponse> {
```

Comment:
I don't think this needs to be a `BaseNodesResponse`, could we make it `ActionResponse`?